docs/library/zephyr.rst

```diff
@@ -0,0 +1,18 @@
+.. currentmodule:: zephyr
+
+:mod:`zephyr` --- functionality specific to the Zephyr port
+===========================================================
+
+.. module:: zephyr
+    :synopsis: functionality specific to Zephyr
+
+The ``zephyr`` module contains functions and classes specific to the Zephyr port.
+
+Classes
```

Comment:
above this subsection you can add a subsection for the functions in `zephyr`: `is_preempt_thread`, `current_tid`, etc